linera-service/tests/linera_net_tests.rs

```diff
@@ -1007,10 +1007,11 @@ async fn test_wasm_end_to_end_non_fungible(config: impl LineraNetConfig) -> Resu
     let nft1_name = "nft1".to_string();
     let nft1_minter = account_owner1;
 
-    let nft1_blob = Blob::test_data_blob("nft1_data");
+    let nft1_blob = Blob::new_data(b"nft1_data".to_vec());
     let nft1_blob_id = nft1_blob.id();
+    let nft1_inner_data = nft1_blob.inner_bytes();
```

Comment:
This test, for example, doesn't use `add_pending_blobs` directly, does it? So it should be able to completely avoid the `Blob` and `BlobId` types?

Reply:
I missed this test file when removing the usages from tests, my bad. Will remove!